<launch>
  <arg name="steer_file" />  
  <arg name="img_dir" />  
  <arg name="bag" />
  <node pkg="rosbag" type="play" name="player" args="--rate=4 --clock $(find f1t)/$(arg bag)"/>

  <!-- <node name="image_saver" pkg="image_view" type="image_saver" output="screen" args="image:=/image_converter/image">
    <param name="filename_format" value="/home/user/catkin_ws/src/f1t/train/image_%04d.%s"/>
  </node>
   -->
  
  <node name="image_converter" pkg="f1t" type="image_converter" output="screen" cwd="node">
    <param name="steer_file" value="$(find f1t)/$(arg steer_file)"/>
    <param name="img_dir" value="$(find f1t)/$(arg img_dir)"/>
  </node>
  
</launch> 